kamon-logback-stackdriver/src/main/scala/kamon/stackdriver/StackdriverMarker.scala

```diff
@@ -0,0 +1,74 @@
+package kamon.stackdriver
+
+import java.util
+
+import kamon.stackdriver.StackdriverMarker._
+import org.slf4j.Marker
+
+import scala.collection.JavaConverters._
+import scala.collection.concurrent.TrieMap
+
+abstract class StackdriverMarker(implicit enc: LogValue.Writer) extends Marker {
```

Comment:
I guess this is a matter of taste, but I would remove this implicit and use writer directly. Otherwise implementation of custom markers based on this one will be more complicated.

Reply:
And that also makes this implementation Scala based, while without too much effort we could make it also extendable with Java.